ts_js_cpp_c/cPart.h

```diff
@@ -0,0 +1,14 @@
+#ifndef FOO_H
+#define FOO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void cPart();
```

Comment:
What do you think to change the funtion `cPart` to be a non `void` function? In this way the example could be more complete and demostrate how to call from ts -> js -> c++ -> c passing some arguments and back.

Reply:
This is a really good suggestion thank you.
What do you think a phrase like this ? :

`I'm a C function and I received this $argument from C++`

And then return an int  back ?

And the the c++ says:

`I'm a C++ function and I received this $argument from JS and this $return from C` 

something like that ? looks cool :smile: 

Since I still learning I'll probably delay a little bit but eager to implement that

Reply:
@NickNaso suggestions applied now it shows the following messages:

```
I'm a C function and I received 12 from C++ and I'm sending back 42 
I'm a C++ function I'm sending 12 to C and I received 42
I'm a Javascript function and I'm sending 11 to C++ and I received 42
I'm a TypeScript class constructor I'm sending 10 to JavaScript and I received 42
```